scripts/iscan

```diff
@@ -209,7 +209,21 @@ cmd_scan() {
 
     local name="$opt_name"
     if [[ -z $name ]]; then
-        name="iscan-$HOSTNAME-$(basename $volume)-$(date --utc +%y%m%d-%H%M%S)"
+        # A tar archive with colon in its name is not easy to unpack --
```

Comment:
See [the context](https://elastio.slack.com/archives/C01UB68QJEA/p1637355543098600?thread_ts=1637353386.096200&cid=C01UB68QJEA).

> The special characters are a pain in the ass.  I think we should remove them.  Make the file easy to work with.  Eliminate `:` and `\` `/` in the name. Still can't get the syntax to work.  I agree, I am not a linux guy but many of our users wont be either.